com.github.sdbg.debug.ui/src/com/github/sdbg/ui/ISDBGLauncher.java

```diff
@@ -0,0 +1,11 @@
+package com.github.sdbg.ui;
+
+import org.eclipse.core.resources.IResource;
+
+public interface ISDBGLauncher {
+
+  String EXTENSION_ID = "com.github.sdbg.debug.ui.sdbgLauncher";
+
+  public void launchChrome(IResource resource, String mode, String url);
```

Comment:
Is there no existing interface that we could use?  An interface on a ChromeLaunchShortcut itself, for example?